member-workers.md

```diff
@@ -0,0 +1,101 @@
+# Member-workers
+
+Hypha has seven **member-workers**, or **members**, all of which are founding members, 
+with six currently serving as directors on the board.
+Members are expected to acknowledge the duties of their relationship with Hypha by signing the *Membership Acknowledgement*.
+
+
+## Being a Good Member
+
+At our [2020 AGM][2020-agm] we began the process of defining what it means to be a good member, we came up with the following initial description:
+
+In order to be a good member, we commit to:
+- being accountable to others, doing the work we've committed to and coming prepared to our initiatives and meetings 
+- making time for each other to learn and grow through co-working and peer sessions 
+- actively soliciting and providing feedback as we work on specific proposals
+- providing confirmations and a sense of closure for others on tasks and initiatives (both synchronously and asynchronously)
+
+Our understanding was shaped from the following prompts and examples:
+
+- Prompts:
+    - _How do you feel accomplished or that you have completed a task?_
+    - _What are the things you do before you come to a meeting?_
+    - _What do you need to (effectively) make a decision together?_
+    - _When do you feel like you've collaborated with hypha members well, what did you do?_
+- What our member statement is / isn't:
+    **IS**:
+        - Skill-based, looking at individual capacities
+        - "We" and "us"-focused (i.e., "I think this will help us work better together")
+    **ISN'T**:
+        - List of working group responsibilities (we have those: https://handbook.hypha.coop/working-groups.html
+        - Managerial ("I think that others need to do...")
+- Examples:
+    > Self-Management is the art of autonomously recognizing what needs to be done, combined with the initiative to work cooperatively, in order to best meeting the collective needs of yourself and the cooperative.
+    - [Casa Nueva](https://institute.coop/sites/default/files/resources/Casa-Nueva-Member-Handbook-2011.pdf)
+    > Careful listening, summarizing and synthesis help us reach a good knowledge and understanding of what everyone needs, and find solutions everyone can accept. Put simply:
+    > - good listening enables us to hear what others are saying;
+    > - summaries help remind us of the key points in the discussion and check we have the same understanding;
+    > - synthesis is the skill that allows us to draw together different views and ideas to form one proposal that works for everyone.
+     - [Seeds for Change "Skills for Consensus"](https://www.seedsforchange.org.uk/consensus#core_skills)
```

Comment:
I feel this could be formatted a bit more clearly in markdown:

- Examples:
  - [insert title]
    > lorem ipsem
  - [Casa Nueva](https://example.com)
    > lorem ipsem
    > - foo
    > - bar
  - [Seeds for Change ...](https://example.com)